driver.sh

```diff
@@ -149,6 +149,20 @@ setup_variables() {
       export ARCH=powerpc
       export CROSS_COMPILE=powerpc64le-linux-gnu- ;;
 
+    "riscv32")
+      config=defconfig
```

Comment:
Should this be `rv32_defconfig`?

Reply:
Probably these two issues are the reason why the kernels produced are nearly identical: https://github.com/ClangBuiltLinux/continuous-integration/pull/70#issuecomment-599708751